lmsweb/views.py

```diff
@@ -275,8 +263,8 @@ def done_checking(solution_id):
     return jsonify({'success': changes.execute() == 1, 'next': next_exercise})
 
 
-def common_comments(exercise_id=None):
-    # TODO: Add cache for 2 minute each time
+@cached(cache=TTLCache(maxsize=TLL_MAXSIZE, ttl=TTL_CACHE_SECONDS))
```

Comment:
Pretty cool, tested and works great 💪

Reply:
> "Please be aware that all these classes are not thread-safe. Access to a shared cache from multiple threads must be properly synchronized, e.g. by using one of the memoizing decorators with a suitable lock object."

This can actually be a huge problem when using Flask :(